<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Encode_Param</class>
 <widget class="QDialog" name="Dialog_Encode_Param">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>分辨率</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_resolution">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>230</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>QOP_STEP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_get_encode_2">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>271</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>获取</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fps_2">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>121</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>211</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>QOP_MAX</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_qop_max_2">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>211</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bitrate_2">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>91</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>151</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>QOP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_method_2">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>61</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>121</y>
     <width>26</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>帧率</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_qop_2">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>151</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>91</y>
     <width>26</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>码率</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_encode_2">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>271</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>181</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>QOP_MIN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_qop_min_2">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>181</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>61</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>编码方式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_qop_step_2">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>241</y>
     <width>129</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
